index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Intuit Accounts - Sign Up</title>
+  </head>
+  <body>
+      <header>
+          <button id="sign_in" class="but"><img src="image/lock.png" alt="lock" class="lock">Sign in</button>
```

Comment:
This image is not showing.

![img](https://user-images.githubusercontent.com/31304943/69757158-8ab4f580-115c-11ea-8a58-c21ed091b299.png)